app.js

```diff
@@ -1,52 +0,0 @@
-
```

Comment:
Looks like you're overwriting your app.js file. Make sure that's what you actually want to do before you merge this PR.